test/test.ts

```diff
@@ -12,7 +12,7 @@ describe("Hello Query Test", function () {
 
   require("../test/hello-test");
   require("../test/new-user-creation-test");
-  require("../test/new-user-error-test");
+  require("../test/login-user-test");
```

Comment:
```suggestion
  require("./login-user-test");
```

Não precisa voltar uma pasta pra depois ir pra mesma novamente. Vale pros requires anteriores tbm.